internal/k8s/crdcontroller.go

```diff
@@ -315,8 +315,23 @@ func checkRefOnController(key, refKey, refValue string) error {
 	}
 
 	if result.Count == 0 {
-		utils.AviLog.Warnf("key: %s, msg: No Objects found for refName: %s/%s", key, refModelMap[refKey], refValue)
-		return fmt.Errorf("%s \"%s\" not found on controller", refModelMap[refKey], refValue)
+		if refModelMap[refKey] == "network" {
+			var rest_response interface{}
+			uri = fmt.Sprintf("/api/%s/%s", refModelMap[refKey], refValue)
+			err := lib.AviGet(clients.AviClient[aviClientLen], uri, &rest_response)
+			if err != nil {
+				utils.AviLog.Warnf("key: %s, msg: Get uri %v returned err %v", key, uri, err)
+				return fmt.Errorf("%s \"%s\" not found on controller", refModelMap[refKey], refValue)
+			} else if rest_response != nil {
+				return nil
```

Comment:
Can we be more verbose about what was the response here and why `rest_response` was `nil`? Maybe also print the `rest_response`?

Reply:
the rest_response is nil when no subnet is found for the given uuid. we can print the response when it is not nil.